index.js

```diff
@@ -4,7 +4,7 @@ var path = require('path');
 
 var commentRx = /^\s*\/(?:\/|\*)[@#]\s+sourceMappingURL=data:(?:application|text)\/json;(?:charset[:=]\S+;)?base64,(.*)$/mg;
 var mapFileCommentRx =
-  // //# sourceMappingURL=foo.js.map                       /*# sourceMappingURL=foo.js.map */
+  /*# sourceMappingURL=foo.js.map */
```

Comment:
Alternatively if you just put a space between them, i.e. `/ /` that'd probably fix Safari as well.